overthebox/files/bin/otb-action-speedtest

```diff
@@ -119,9 +119,12 @@ setup
 # Cleanup getting a SIGINT/HUP/TERM signal
 trap : HUP INT TERM
 
+INPUT=$1
+echo "$1" | jq -e . 1>/dev/null 2>/dev/null && INPUT=
```

Comment:
`[ -d /sys/class/net/$1 ] && INPUT=$1`

Reply:
The `$1` argument is an interface name, not a device name